src/libtriton/bindings/python/namespaces/initSolverNamespace.cpp

```diff
@@ -38,7 +38,11 @@ namespace triton {
       void initSolverNamespace(PyObject* solverDict) {
         PyDict_Clear(solverDict);
 
+        #if defined(TRITON_Z3_INTERFACE)
         xPyDict_SetItemString(solverDict, "Z3", PyLong_FromUint32(triton::engines::solver::SOLVER_Z3));
+        #elif defined(TRITON_BITWUZLA_INTERFACE)
+        xPyDict_SetItemString(solverDict, "BITWUZLA", PyLong_FromUint32(triton::engines::solver::SOLVER_BITWUZLA));
+        #endif
```

Comment:
I'm not sure it's an `elif` here. What if z3 and Bitwuzla are both compiled ? Should not be two `ifdef` ?

Reply:
Fixed that bug